javascript/organizing_your_javascript_code/es6_modules.md

```diff
@@ -130,7 +128,9 @@ document.body.appendChild(component());
 Easy! Now, if you run `npx webpack` in your project directory, your page should show our new function being used.
 
 <span id="exports-knowledge-check"></span> 
-There are 2 different ways to use exports in your code: named exports and default exports. Which option you use depends on what you're exporting. As a general rule if you want to export multiple functions use named exports with this pattern:
+There are two different ways to use exports in your code: named exports and default exports. Which option you use depends on what you're exporting.vTake a moment to look at the [MDN documentation about the import declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and the [MDN documentation about the export declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) which give great examples for the different syntax on import and export.
+
+Here is an example of use named exports which you will most often use when you have multiple values in a module to export:
```

Comment:
```suggestion
Here is an example of with named exports, which you will most often use when you have multiple values to export in a module:
```
A slight rephrasing, since one may want to stress on the fact that you are "Exporting multiple values"